src/test/java/spoon/test/record/CtRecordTest.java

```diff
@@ -146,6 +146,16 @@ private CtModel createModelFromPath(String code) {
 		return model;
 	}
 
+	@Test
+	void printGenericRecord() {
```

Comment:
We try to name new tests we create by describing what is happening in the test. I think a better name would be `testGenericTypeParametersArePrintedBeforeTheFunctionParameters`.

Reply:
OK, will update the PR